### ISSUE TITLE
Importing decimal library causes error

### ISSUE THREAD

**Issue by TheCleric** [Microsoft/pyright#43]
**Describe the bug**
When importing the python standard decimal library, an error is given: `An internal error occurred while performing type analysis`

**To Reproduce**
Have a file where `import decimal` is present.

**Expected behavior**
No error for standard libraries

**Screenshots or Code**
```
import decimal
```

**VS Code extension or command-line**
VSCode extension

**Additional context**
Python version: 3.6.5 (Anaconda)

**Comment by erictraut** [comment]
Thanks for the bug report. I'm not able to repro this behavior. Here are a few follow-up questions for you.

What version of the VSCode extension are you running? You can find it by selecting Pyright in the extensions panel.

Are you able to repro this with a source file that includes nothing other than "import decimal", or is there additional code in the file?

What version of the decimal library do you have installed?

**Comment by TheCleric** [comment]
> What version of the VSCode extension are you running? You can find it by selecting Pyright in the extensions panel.

1.0.7

> Are you able to repro this with a source file that includes nothing other than "import decimal", or is there additional code in the file?

Good point. Was not able to reproduce with just import decimal, but here's the minimum code from my original code that still produces the error.

```python
import decimal
from typing import Dict


def _make_dynamo_update_from_dict(obj: Dict):
    def prepare_value(val):
        if isinstance(val, float):
            return decimal.Decimal(val)
        return val

    object_dict = {
        ":{}".format(k): prepare_value(v) for k, v in obj.items()
        if v is not None
    }

    expression_names = {}
    statement_pairs = [
        "{} = {}".format(k[1:], k)
        for k, v in object_dict.items()
    ]

    update_expression = "set {}".format(
        ", ".join(statement_pairs)
    )

    return update_expression, object_dict, expression_names
```

> What version of the decimal library do you have installed?

The default one with python. Though when trying to command click into it, VSCode/pyright may be getting it confused with one from pyright. See screenshot attached.
![](https://i.imgur.com/BOX9oyy.png)

**Comment by erictraut** [comment]
Thanks for the bug report. I found and fixed the problem. The fix will be in the next release (1.0.8).

**Comment by erictraut** [comment]
I just published 1.0.8. Please re-open if you still see any issues.
